### PR TITLE
Remove API code, add references to other GitHub repositories

### PR DESCRIPTION
**Pull request by Nakaner**
We have switched to the new API. Threfore the old code can be deleted.

### CODE REVIEW

**Comment by DerDakon** [comment]
You need to update INSTALL.md as well I think as it talks about the APi.

File: INSTALL.md [hunk 1]
```diff
@@ -55,19 +54,19 @@ Install Leaflet and the Leaflet extension that provides the edit link:
...
+$ tar -xvzf 1.9.2.tar.gz
 $ mkdir osm2pgsql-build
 $ cd osm2pgsql-build
-$ cmake ../osm2pgsql-0.96.0
+$ cmake ../osm2pgsql-1.9.2
 $ make
 $ sudo make install
 $ cd ..
-$ rm -fr osm2pgsql-build osm2pgsql-0.96.0 0.96.0.tar.gz
+$ rm -fr osm2pgsql-build osm2pgsql-1.9.2 1.9.2.tar.gz
```
Comment (DerDakon) [inline INSTALL.md:69]: I would omit the `rm` here as well as in other places. Everyone should be able to clean that up alone if it is relevant.

File: INSTALL.md [hunk 2]
```diff
@@ -398,17 +362,21 @@ CustomLog /var/log/apache2/tiles.access.log combined
...
-mkdir /var/lib/tirex/tiles/standard
-mkdir /var/lib/tirex/tiles/maxspeed
-mkdir /var/lib/tirex/tiles/signals
-mkdir /var/lib/tirex/tiles/electrification
+mkdir /var/cache/tirex/tiles/standard
+mkdir /var/cache/tirex/tiles/maxspeed
+mkdir /var/cache/tirex/tiles/signals
+mkdir /var/cache/tirex/tiles/electrification
+mkdir /var/cache/tirex/tiles/gauge
+for STYLE in standard maxspeed signals electrification gauge ; do
```
Comment (DerDakon) [inline INSTALL.md:370]: put the mkdir in the loop

**Review by DerDakon — CHANGES_REQUESTED** [review]

File: INSTALL.md [hunk 3]
```diff
@@ -132,19 +110,19 @@ Clone the server configuration repository:
 ```shell
 mkdir /srv/OpenRailwayMap-setup
 git clone https://github.com/OpenRailwayMap/server-admin.git /srv/OpenRailwayMap-setup/
-cd /srv/OpenRailwayMap-setup/scripts
+cd /srv/OpenRailwayMap-setup/ansible/role/tileserver/files/scripts
```
Comment (vhscom) [inline INSTALL.md:113]: Thanks for updating this. Had me scratching my head last week. Eventually figured it out. Now we won't need to.

Comment (vhscom) [inline INSTALL.md:113]: Thanks for fixing this. Had me scratching my head last week.

**Pull request merged**